### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by ritazakhodyaeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -5,7 +5,9 @@
...
 
+Pivot Grid does not display field values in row and column areas if these values are not represented in the underlying data source. This example demonstrates how to create a data source wrapper that contains all required values on its level, and there is no need to modify original data source. In the example, the `CustomDates` collection is filled with DateTime values based on the actual Start/End DateTime range and the specified interval (for example, `Month`). This collection is passed to the `DateDataSourceWrapper` class instance along with the original data source collection (`Table.DefaultView`). `CustomDates` collection is merged with the original data table and you do not need to modify the data source. When Pivot Grid requests data with the common [IList](https://msdn.microsoft.com/en-us/library/system.collections.ilist(v=vs.110).aspx) and [ITypedList](https://msdn.microsoft.com/en-us/library/system.componentmodel.itypedlist(v=vs.110).aspx) interfaces methods, the `DateDataSourceWrapper` object returns data from the original collection or generates `EmptyObjectPropertyDescriptor` objects to return rows with null "Date" field values.
```
Comment (albertov05) [inline Readme.md]: 1. `This example demonstrates how to create a data source wrapper that contains all required values on its level, and there is no need to modify original data source.`

a) What do you mean by "on its level"?
b) Please change "modify original data source" to "modify the  original data source".

2. ``CustomDates` collection is merged with the original data table and you do not need to modify the data source.`

Please change "`CustomDates` collection" to "The `CustomDates` collection".

3. `When Pivot Grid requests data with the common [IList](https://msdn.microsoft.com/en-us/library/system.collections.ilist(v=vs.110).aspx) and [ITypedList](https://msdn.microsoft.com/en-us/library/system.componentmodel.itypedlist(v=vs.110).aspx) interfaces methods`...

Please change "interfaces methods" to "interface methods".

File: Readme.md [hunk 2]
```diff
@@ -5,7 +5,9 @@
...
 
+The following image illustrates Pivot Grid that displays the custom `March` field value from the `CustomDates` collection along with field values from the data source:
```
Comment (albertov05) [inline Readme.md]:
```suggestion
The following image illustrates a Pivot Grid that displays the custom `March` field value from the `CustomDates` collection along with field values from the data source:

```

File: Readme.md (hunk 1 above)
Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
Pivot Grid does not display field values in row and column areas if these values are not represented in the underlying data source. This example demonstrates how to create a data source wrapper that contains merged values ​​from a data source and a custom collection. In the example, the `CustomDates` collection is filled with DateTime values based on the actual Start/End DateTime range and the specified interval (for example, `Month`). This collection is passed to the `DateDataSourceWrapper` class instance along with the original data source collection (`Table.DefaultView`). The `CustomDates` collection is merged with the original data table and you do not need to modify the data source. When Pivot Grid requests data with the common [IList](https://msdn.microsoft.com/en-us/library/system.collections.ilist(v=vs.110).aspx) and [ITypedList](https://msdn.microsoft.com/en-us/library/system.componentmodel.itypedlist(v=vs.110).aspx) interface methods, the `DateDataSourceWrapper` object returns data from the original collection or generates `EmptyObjectPropertyDescriptor` objects to return rows with null "Date" field values.

```

Comment (albertov05) [reply Readme.md]: Thank you!

Comment (albertov05) [reply Readme.md]: Please apply your suggestion.

**Pull request merged**